nanoFramework.Device.Bluetooth/GenericAttributeProfile/GattServiceProvider.cs

```diff
@@ -35,11 +37,48 @@ public sealed class GattServiceProvider
 
         internal GattServiceProvider(Guid serviceUuid)
         {
-            _service = new GattLocalService(serviceUuid);
+            _services = new ArrayList();
+
+            // Add primary
+            AddService(serviceUuid);
+
+            // Add default Device Information service 
+            AddDeviceInformationService();
 
             NativeInitService();
         }
 
+        /// <summary>
+        /// Add default Device Information Service
+        /// </summary>
+        private void AddDeviceInformationService()
+        {
+            // Add and initialised Device Information defaults
+            GattLocalService dinfService = AddService(GattServiceUuids.DeviceInformation);
+
+            // ManufacturerNameString Characteristic (0x2A29)
+            DataWriter manufacturerName = new DataWriter();
+            manufacturerName.WriteString("Nanoframework");
```

Comment:
Maybe we should follow the correct branding?

```suggestion
            manufacturerName.WriteString("nanoFramework");
```